package.json

```diff
@@ -35,24 +40,28 @@
     "html-webpack-plugin": "2.29.0",
     "jest": "20.0.4",
     "material-components-web": "^0.43.0",
-    "node-sass": "^4.8.3",
+    "node-sass": "^4.11.0",
```

Comment:
FYI this is going to conflict with master now, since Abhinay switched it to dart-sass. You might want to sync and re-test.

Reply:
Will be handled by merging with master